modules/core/src/main/scala/org/scalasteward/core/nurture/NurtureAlg.scala

```diff
@@ -123,7 +123,8 @@ final class NurtureAlg[F[_]](
   def createPullRequest(data: UpdateData): F[Unit] =
     for {
       _ <- logger.info(s"Create PR ${data.updateBranch.name}")
-      requestData = NewPullRequestData.from(data, config.vcsLogin)
+      branchName = vcsSpecifics.headForListingPullRequests(data.fork, data.update)
+      requestData = NewPullRequestData.from(data, branchName, config.vcsLogin)
```

Comment:
This will use `owner/repo:update/dependency-1.2.0` as `branchName` in case of GitHub but it should be `owner:update/dependency-1.2.0`. It is unfortunate that GitHub requires two different `head` parameters for listing pull requests and creating new ones.

Reply:
You are right, what I wanted here is `owner:update/dependency-1.2.0`. I've fixed it with a second method in VCS Specifics.

But maybe instead of using the interface VCSSpecifics that I added, we could replace it with functions in package object (like it was before) but that takes in the `VCSType` ?

Reply:
Sounds good to me too.